From 7c778820670ea29c7e460c106f59038f8a8fa8ae Mon Sep 17 00:00:00 2001
From: Zaikuo Wang <zaikuo.wang@intel.com>
Date: Thu, 4 Jun 2015 11:07:30 +0800
Subject: [PATCH 411/462] ov2722: remove 720p setting in video mode

Because ov2722 has no scaling function, the 720p resolution setting
which crop from full size has FOV reduction issue

Change-Id: I20568fa7c1dda2bac572215f981adc300bda0a55
Track-On: https://jira01.devtools.intel.com/browse/IMINAN-33103
Signed-off-by: Zaikuo Wang <zaikuo.wang@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/58861
Build: IMIN (SC)
Approver: Yunliang Ding <yunliang.ding@intel.com>
Reviewed-by: Yunliang Ding <yunliang.ding@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../external_drivers/camera/drivers/media/i2c/ov2722.h   | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h b/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
index 7621608..fde004e 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
@@ -1260,22 +1260,6 @@ struct ov2722_resolution ov2722_res_video[] = {
 		.regs = ov2722_480P_30fps,
 	},
 	{
-		.desc = "ov2722_720p_30fps",
-		.width = 1296,
-		.height = 736,
-		.fps = 30,
-		.pix_clk_freq = 75,
-		.used = 0,
-		.pixels_per_line = 2048,
-		.lines_per_frame = 1187,
-		.bin_factor_x = 1,
-		.bin_factor_y = 1,
-		.bin_mode = 0,
-		.skip_frames = 3,
-		.regs = ov2722_720p_30fps,
-		.mipi_freq = 364800,
-	},
-	{
 		.desc = "ov2722_1080P_30fps",
 		.width = 1932,
 		.height = 1092,
-- 
1.9.1

